docker/cla_public.ini

```diff
@@ -11,8 +11,6 @@ processes = 2
 chdir = /home/app/flask
 module = cla_public.wsgi
 callable = app
-req-logger = file:/var/log/wsgi/request.log
-logger = file:/var/log/wsgi/error.log
```

Comment:
Would specifying `file:/dev/stdout` work?

Reply:
No, that doesn't seem to work. If those lines are removed, uwsgi logs to stdout okay. Here you can see 2 logs entries, the first one is from uwsgi and the second from nginx.

```
localhost:5000 [pid: 36|app: 0|req: 10/14] 192.168.144.1 () {52 vars in 1356 bytes} [Tue Sep 18 15:00:52 2018] GET /scope/diagnosis/ => generated 20624 bytes in 144 msecs (HTTP/1.1 200) 5 headers in 500 bytes (1 switches on core 0)
{ "@timestamp": "2018-09-18T15:00:52+01:00", "@fields": { "remote_addr": "192.168.144.1", "remote_user": "-", "body_bytes_sent": 6503, "request_time": 0.145, "request_uri": "/scope/diagnosis/", "status": 200, "request": "GET /scope/diagnosis/ HTTP/1.1", "request_method": "GET", "http_referrer": "-", "http_user_agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/68.0.3440.106 Safari/537.36", "bytes_sent": 7197, "gzip_ratio": "3.18", "http_host": "localhost", "sent_http_location": "-", "server_name": "~^(.+)$", "server_port": "80", "upstream_addr": "unix:////tmp/public.sock", "upstream_response_length": "20624", "upstream_response_time": "0.140", "upstream_status": "200" } }
```

I need to customise the logs as per the ticket JSON.

Reply:
👍 

:badpokerface: In #755, I ended up converting the `wsgi` logs to logstash format as part of simplifying the configuration -- so rebasing might change the behaviour. Sorry :(